mafia/data/raretiles.json

```diff
@@ -220,6 +224,8 @@
   { "minute": 2566, "row": 10, "column": 0 },
   { "minute": 2568, "row": 8, "column": 3 },
   { "minute": 2577, "row": 4, "column": 4 },
+  { "minute": 2592, "row": 10, "column": 3 },
+  { "minute": 2605, "row": 7, "column": 5 },
```

Comment:
You found one of the 34 new rares that I have discovered.
The other 9 you have found are new to me.

I'm going to commit my discoveries to BeachComber, soon.
After we've been through tides=0 again, in 2 days.

For now, I have added your discoveries to my "known" rare lists.

Reply:
Yeah that's fine, I will note that I too plan to hit the beaches on tides 0, which is probably going to be 7.5 to 10k

May the fastest win.
Though I will also note that my script will probably hit them only once each, so if there's more than one sparkle..
![image](https://github.com/loathers/combo/assets/1419679/ed06f972-d320-433d-a0d1-3cc1ca0482f4)

Reply:
Well, I found nothing new